pynxtools/dataconverter/readers/xrd/Untitled.ipynb

```diff
@@ -0,0 +1,844 @@
+{
```

Comment:
Notebooks and data files should be removed or moved to example folder.

Reply:
This reader would be extended to other file versions as well. So, I want to keep it, so that I do not need to create some test script again and again. But definitely, It would not be there for final version.

Reply:
Hm okay, but can we move them to example maybe? It doesn't feel right to leave it in dataconverter. And the notebooks could be cleaned a bit (can be done later though).

Reply:
It is not there anymore.